src/ext/graphql/transmodelapi/schema.graphql

```diff
@@ -715,6 +715,8 @@ type QueryType {
     dateTime: DateTime,
     "Debug the itinerary-filter-chain. OTP will attach a system notice to itineraries instead of removing them. This is very convenient when tuning the filters."
     debugItineraryFilter: Boolean = false,
+    "A list of filters for which trips should be included. A trip will be included if it matches with at least one filter. An empty list of filters means that all trips should be included. If a search include this parameter, \"whiteListed\", \"banned\" & \"modes.transportModes\" filters will be ignored."
```

Comment:
What should happen to the `whiteListed`, `banned` and `modes.TransportModes` params? Should they be deprecated?

Reply:
The `whiteListed` and `banned` should be deprecated - not so sure about TransportModes.

Reply:
Everything can be defined through new filter API. I think those three should be deprecated.